samples/bluetooth/hap_ha/prj.conf

```diff
@@ -0,0 +1,54 @@
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_SMP=y
+CONFIG_BT_SMP_SC_PAIR_ONLY=y
+CONFIG_BT_SMP_MIN_ENC_KEY_SIZE=16
+
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_DEVICE_NAME="Hearing Aid sample"
+CONFIG_BT_DEVICE_APPEARANCE=2625
+
+CONFIG_BT_AUDIO=y
+CONFIG_BT_AUDIO_UNICAST_SERVER=y
+CONFIG_BT_ASCS_ASE_SNK_COUNT=1
+CONFIG_BT_ASCS_ASE_SRC_COUNT=1
+# Support an ISO channel per ASE
+CONFIG_BT_ISO_MAX_CHAN=1
```

Comment:
Hmm, you have right. The HAP mandates bidirectional, that's why 1 ISO channel. So the comment here is invalid. But anyway, 2 channels should be supported anyway in case the Unicast Client would try to create unidirectional channels. So I'll change it to 2

Reply:
One thing to keep in mind:
An ASE is unidirectional, and that 2 ASEs can use the same bidirectional stream (assuming they are a sink and source ASE). 

The relationsship between `CONFIG_BT_ISO_MAX_CHAN` and number of ASEs is pretty loose actually because of this, and the fact that 1 ASE can use up to `CONFIG_BT_MAX_CONN` ISO streams since ASEs are _per connection_ and `CONFIG_BT_ISO_MAX_CHAN` is not. 

IMO it's fine that `CONFIG_BT_ISO_MAX_CHAN=1`, but the comment just stated something different.